confbot_driver/src/confbot_actionclient.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <inttypes.h>
+#include <memory>
+
+#include "confbot_msgs/action/move_command.hpp"
+
+#include "rclcpp/rclcpp.hpp"
+#include "rclcpp_action/rclcpp_action.hpp"
+
+using MoveCommand = confbot_msgs::action::MoveCommand;
+
+void feedback_callback(
+  rclcpp_action::ClientGoalHandle<MoveCommand>::SharedPtr,
+  const std::shared_ptr<const MoveCommand::Feedback> feedback)
+{
+  RCLCPP_INFO(
+    rclcpp::get_logger("feedback logger"),
```

Comment:
Nit: maybe use a more descriptive name that mentions this is tied to the confbot action client